assets/js/session/index.js

```diff
@@ -522,6 +530,25 @@ function saveNotebook(hook) {
   hook.pushEvent("save", {});
 }
 
+function captureCodeSnippet(hook) {
+  if (hook.state.focusedCellId) {
+    html2canvas(
+      document.querySelector("#cell-" + hook.state.focusedCellId)
```

Comment:
There's `getCellById` helper so you can do `getCellById(hook.state.focusedCellId)`. Though I think this way we show the action buttons as well, right?

Reply:
@jonatanklosko I believe we can hide it. Used the helper.

I implemented your suggestions and omitted the action buttons.

Does that look better to you?